assets/stylesheets/custom.css.scss

```diff
@@ -28,3 +38,30 @@ body {
 .panel-body {
   font-size: 1.33em;
 }
+
+.text-center h1 {
+  font-size: 4.5em;
+  margin: 55px 0;
+}
```

Comment:
The default `h1` size is good. Keep in mind that when you change something this high, it will affect all instances of it.